.gitignore

```diff
@@ -223,5 +223,8 @@ Pipfile
 Pipfile.lock
 
 .DS_Store
+
 .history
 .vscode
+
+downloads
```

Comment:
the following
https://github.com/brighter-ai/redact-client/blob/648d0741f12b1949f878e9f6fbc89c1d65636892/.gitignore#L89
is not sufficient for symlinks, added it to avoid that behavior